src/api/RestAPI.ts

```diff
@@ -244,6 +244,7 @@ export class RestAPI extends EventEmitter {
         const params: RequestInit = {
             headers: this.getHeaders(data, auth, payload, acceptType),
             method,
+            credentials: "include",
```

Comment:
this seems to be the only change we need on the frontend

Reply:
We should probably use the `auth` parameter to set this 🙂 Testing all the cases may take a while otherwise.